io.freetubeapp.FreeTube.metainfo.xml

```diff
@@ -3,15 +3,411 @@
+  <summary xml:lang="hi">गोपनीयता के लिए एक ओपन-सोर्स यूट्यूब ऐप</summary>
+  <summary xml:lang="hr">YouTube aplikacija otvorenog koda za privatnost</summary>
+  <summary xml:lang="hu">Nyílt forráskódú YouTube-alkalmazás az adatvédelem érdekében</summary>
+  <summary xml:lang="hy">Բաց կոդով YouTube հավելված՝ գաղտնիության համար</summary>
+  <summary xml:lang="id">Aplikasi YouTube sumber terbuka untuk privasi</summary>
+  <summary xml:lang="is">Opinn YouTube app fyrir friðhelgi einkalífsins</summary>
+  <summary xml:lang="it">Un'app YouTube open source per la privacy</summary>
+  <summary xml:lang="ja">プライバシー保護のためのオープンソースのYouTubeアプリ</summary>
+  <summary xml:lang="kn">ಗೌಪ್ಯತೆಗಾಗಿ ಓಪನ್ ಸೋರ್ಸ್ YouTube ಅಪ್ಲಿಕೇಶನ್</summary>
+  <summary xml:lang="ko">개인 정보 보호를 위한 오픈 소스 YouTube 앱</summary>
+  <summary xml:lang="lt">Atvirojo kodo „YouTube“ programėlė privatumui užtikrinti</summary>
+  <summary xml:lang="lv">Atvērtā koda YouTube lietotne privātuma nodrošināšanai</summary>
+  <summary xml:lang="nl">Een open-source YouTube-app voor privacy</summary>
+  <summary xml:lang="no_NY">En YouTube-app med åpen kildekode for personvern</summary>
+  <summary xml:lang="oc">Una aplicacion YouTube de còde dobèrt per la vida privada</summary>
+  <summary xml:lang="pa">ਗੋਪਨੀਯਤਾ ਲਈ ਇੱਕ ਓਪਨ-ਸੋਰਸ YouTube ਐਪ</summary>
+  <summary xml:lang="pl">Aplikacja YouTube z otwartym kodem źródłowym zapewniająca prywatność</summary>
+  <summary xml:lang="pt_BR">Um aplicativo de código aberto do YouTube para privacidade</summary>
+  <summary xml:lang="pt">Uma aplicação de código aberto do YouTube para privacidade</summary>
+  <summary xml:lang="ro">O aplicație YouTube open-source pentru confidențialitate</summary>
+  <summary xml:lang="ru">Приложение YouTube с открытым исходным кодом для обеспечения конфиденциальности</summary>
+  <summary xml:lang="si">පෞද්ගලිකත්වය සඳහා විවෘත මූලාශ්‍ර YouTube යෙදුමක්</summary>
+  <summary xml:lang="sk">Aplikácia YouTube s otvoreným zdrojovým kódom pre ochranu súkromia</summary>
+  <summary xml:lang="sl">Odprtokodna aplikacija YouTube za zasebnost</summary>
+  <summary xml:lang="sq">Një aplikacion YouTube me burim të hapur për privatësi</summary>
+  <summary xml:lang="sr">YouTube апликација отвореног кода за приватност</summary>
+  <summary xml:lang="sv">En YouTube-app med öppen källkod för integritet</summary>
+  <summary xml:lang="ta">தனியுரிமைக்கான ஓப்பன் சோர்ஸ் YouTube பயன்பாடு</summary>
+  <summary xml:lang="tr">Gizlilik için açık kaynaklı bir YouTube uygulaması</summary>
+  <summary xml:lang="uk">Додаток YouTube з відкритим кодом для забезпечення конфіденційності</summary>
+  <summary xml:lang="vi">Một ứng dụng YouTube nguồn mở bảo vệ quyền riêng tư</summary>
+  <summary xml:lang="zh_CN">一款保护隐私的开源 YouTube 应用</summary>
+  <summary xml:lang="zh_TW">一款保護隱私的開源 YouTube 應用</summary>
+  <summary xml:lang="zu">Uhlelo lokusebenza lwe-YouTube olunomthombo ovulekile lobumfihlo</summary>
```

Comment:
I dont see this anywhere on https://flathub.org/languages. Remove all translations related to this lang